airbyte-webapp/src/packages/cloud/services/dbtCloud.ts

```diff
@@ -25,47 +31,35 @@ import { useUpdateWorkspace } from "services/workspaces/WorkspacesService";
 import { useConfig } from "./config";
 
 export interface DbtCloudJob {
-  account: string;
-  job: string;
+  accountId: number;
+  jobId: number;
   operationId?: string;
   jobName?: string;
 }
```

Comment:
@ambirdsall I wasn't quite sure if we even need this type anymore. Could we maybe just use `OperatorWebhookDbtCloud` from the Orval generated `AirbyteClient.ts`?